src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/PointsToAnalysis/PointsToAnalysis.PointsToDataFlowOperationVisitor.cs

```diff
@@ -255,11 +294,30 @@ protected override void PostProcessArgument(IArgumentOperation operation, bool i
             {
                 base.PostProcessArgument(operation, isEscaped);
 
-                if (!isEscaped &&
-                    operation.Parameter.RefKind != RefKind.None &&
-                    AnalysisEntityFactory.TryCreate(operation, out var analysisEntity))
+                if (!isEscaped)
+                {
+                    // Update abstract value for unescaped ref or out argument (interprocedural analysis case).
+                    if (operation.Parameter.RefKind != RefKind.None &&
```

Comment:
Do you have to worry about RefKind.In here?

Reply:
Good point, I should update the check to match the comment. Given that the abstract value for `in` arguments cannot change after the invocation, we should probably be fine either way.